docs/pywbemclifeatures.rst

```diff
@@ -289,20 +289,44 @@ The following are examples of scalar property definitions:
 Interactively selecting INSTANCENAME
 ------------------------------------
 
-The INSTANCENAME argument has a certain complexity, particularly for
-associations and for classes with multiple keys.
+The INSTANCENAME argument can be complex, particularly for associations and
+classes with multiple keys.
+
+To simplify using INSTANCENAME on the command line, pywbemcli provides a wild
+card character "?" that can be used with commands in place of the INSTANCENAME
+keys on commands that have an INSTANCENAME argument. This allows the user to
```

Comment:
Since we are scoping the sentence already to INSTANCENAME as per its first words, I think we can remove " on commands that have an INSTANCENAME argument".

Reply:
DONE